packages/device_info_plus/device_info_plus/example/integration_test/device_info_plus_test.dart

```diff
@@ -67,6 +67,17 @@ void main() {
     }
   });
 
+  testWidgets('Can get non-null iOS utsname fields',
+      (WidgetTester tester) async {
+    if (Platform.isIOS) {
+      expect(iosInfo.utsname.machine, 'iPhone10,4');
+      expect(iosInfo.utsname.nodename, isNotNull);
+      expect(iosInfo.utsname.release, isNotNull);
+      expect(iosInfo.utsname.sysname, isNotNull);
+      expect(iosInfo.utsname.version, isNotNull);
+    }
+  });
```

Comment:
Can you make the following changes:

- remove the if condition
- add `skip: !Platform.isIOS` to the test

Reply:
Updated 👍